docs/team/ngdeqi.md

```diff
@@ -0,0 +1,49 @@
+### Project: Property Rental Manager
+
+Property Rental Manager (PRM) is a desktop application that helps property agents manage, filter and monitor single
+owner rental units for appropriate tenants. This application uses Command Line Interface (CLI) and is able to display
+information quickly with minimal latency.
+
+Here are my contributions to the project:
+
+* **New Feature**: Added the ability to pair properties with clients.
+    * What it does: Allows the user to record the event where a client rents a property, by pairing them together.
+    * Justification: This feature helps the user track which client(s) is/are renting which property.
+
+* **New Feature**: Added the ability to unpair properties from clients.
+  * What it does: Allows the user to record the event where a client is no longer renting a property, by unpairing them.
+  * Justification: This feature helps the user track which client(s) is/are renting which property.
+
+* **New Feature**: Added the ability to check the information of properties.
+  * What it does: Allows the user to view all information relating to a property, including the tenants who are renting it.
+  * Justification: This feature allows the user to easily pull up and view all information related to a specific property.
+
+
+
+* **Code contributed**: [RepoSense link](https://nus-cs2113-ay2223s1.github.io/tp-dashboard/?search=&sort=groupTitle&sortWithin=title&timeframe=commit&mergegroup=&groupSelect=groupByRepos&breakdown=true&checkedFileTypes=docs~functional-code~test-code~other&since=2022-09-16&tabOpen=true&tabType=authorship&tabAuthor=ngdeqi&tabRepo=AY2223S1-CS2113-F11-1%2Ftp%5Bmaster%5D&authorshipIsMergeGroup=false&authorshipFileTypes=docs~functional-code~test-code&authorshipIsBinaryFileTypeChecked=false&authorshipIsIgnoredFilesChecked=false)
+
+* **Project management**:
```

Comment:
Similar to what @wilsonngja suggested on the consistency of headers, do consider standardizing their format (first alphabet of every word capitalized).